File: lc-blind75/Twosum(lc-1)/Bruteforce/Solution.java
import java.util.*;
public class Solution {
    public static void main(String args[]){
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
        int nums[]=new int [n];
        for(int i=0;i<n;i++){
            nums[i]=sc.nextInt();
        }
        int target=sc.nextInt();
        int res[]=twoSum(nums,target);
        for(int i=0;i<2;i++){
            System.out.println(res[i]);
        }
        sc.close();
    }
    public static int[] twoSum(int nums[],int target){
        int n=nums.length;
        int res[]=new int [2];
        for(int i=0;i<n;i++){
            for(int j=i+1;j<n;j++){
                if(nums[i]+nums[j]==target){
                    res[0]=i;
                    res[1]=j;
                }
            }
        }
        return res;
    }
}

File: lc-blind75/Twosum(lc-1)/Bruteforce/Solution.py
def twoSum(nums,target):
    res=[]
    n=len(nums)
    for i in range(n):
        for j in range(i+1,n):
            if nums[i]+nums[j] ==target:
                res.append(i)
                res.append(j)

    return res

n=int(input())
nums=list(map(int,input().split()))
target=int(input())
print(twoSum(nums,target))